ci/do_ci.sh

```diff
@@ -4,7 +4,6 @@
 
 set -e
 
-
```

Comment:
these changes seem unrelated to this pr - please revert

Reply:
Accidentally this file got pushed while merging main branch. Only realized after you pointed. Thanks a lot!